CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Honeycomb Lambda Extension Changelog
 
-## 10.2.0 (2002-04-08)
+## [v10.2.0] Layer 9 - 2022-04-08
```

Comment:
Layer version?
```suggestion
## [v10.2.0] Layer version 9 - 2022-04-08
```

Reply:
Yeah, I couldn't decide what was best, it's a bit weird. I updated to include the word "version" in https://github.com/honeycombio/honeycomb-lambda-extension/pull/83/commits/4a0aa5017bbde355de408c14de83ea9c75dc354b